frontend/src/components/details/top_card.svelte

```diff
@@ -37,7 +45,11 @@
             {#if genres}
                 <div class="flex-wrap mt-2">
                     {#each genres as genre}
-                        <div class="badge mx-2">{genre}</div>
+                        <div
+                            on:click={() => changeGenreAndRedirectHome(genre)}
+                            class="badge mx-2 transform hover:scale-110 cursor-pointer">
```

Comment:
I think we should make the badges use `primary`-color to indicate that they are clickable - I think that's how the pro frontenders do it 🙈